global.json

```diff
@@ -1,9 +1,11 @@
 {
   "sdk": {
+    "version": "9.0.200-preview.0.24518.5",
+    "rollForward": "major",
     "allowPrerelease": true
   },
   "tools": {
-    "dotnet": "9.0.100-rc.2.24474.11",
+    "dotnet": "9.0.200-preview.0.24518.5",
```

Comment:
I wouldn't update the toolset if we can avoid it, I'd rather not be on such an early prerelease.